template/App.tsx

```diff
@@ -62,19 +61,25 @@ function App(): React.JSX.Element {
     backgroundColor: isDarkMode ? Colors.darker : Colors.lighter,
   };
 
+  // Note: adding hacky padding to avoid importing rn-safe-area-context library but should not be done in production
+  const hackyPadding = '5%';
```

Comment:
```suggestion
  /*
   * To keep the template simple and small we're adding padding to prevent view 
   * from rendering under the System UI.
   * For bigger apps the reccomendation is to use `react-native-safe-area-context`:
   * https://github.com/AppAndFlow/react-native-safe-area-context
   *
   * You can read more about it here:
   * https://github.com/react-native-community/discussions-and-proposals/discussions/827
   */
  const safePadding = '5%';
```